gadget-container/entrypoint.sh

```diff
@@ -80,16 +80,20 @@ if [ "$HOOK_MODE" = "crio" ] ; then
     cp /opt/hooks/oci/$i /host/opt/hooks/oci/
   done
 
-  echo "Installing OCI hooks configuration in /etc/containers/oci/hooks.d"
-  mkdir -p /host/etc/containers/oci/hooks.d
-  cp /opt/hooks/crio/gadget-prestart.json /host/etc/containers/oci/hooks.d/ 2>/dev/null || true
-  cp /opt/hooks/crio/gadget-poststop.json /host/etc/containers/oci/hooks.d/ 2>/dev/null || true
-
-  if ! ls /host/etc/containers/oci/hooks.d/gadget-{prestart,poststop}.json > /dev/null 2>&1; then
-    echo "Couldn't install OCI hooks configuration" >&2
-  else
-    echo "Hooks installation done"
-  fi
+  for HOOK_PATH in "/host/etc/containers/oci/hooks.d" \
+                   "/host/usr/share/containers/oci/hooks.d/"
+  do
```

Comment:
So, it means that minikube with CRI-O uses a different path than OpenShift (because it is working). Do you think adding a test in the integration test for this configuration is worth adding?

Reply:
Yes, those use different folders and that's why it wasn't working on minikube. I think adding an integration test for it would be great, but I don't know how difficult it's. Created https://github.com/inspektor-gadget/inspektor-gadget/issues/1098 to track this down.